src/indigenousLayers.js

```diff
@@ -38,49 +37,35 @@ L.LayerGroup.IndigenousLandsTerritoriesLayer = L.LayerGroup.extend(
                 (function() {
                     var zoom = self._map.getZoom(), origin = self._map.getCenter() ;
                     var $ = window.jQuery;
+                    var ILL_url;
```

Comment:
```suggestion
                    var ILL_url = this.name === "Territories" || this.name === "Languages" ? "https://native-land.ca/api/index.php?maps="+this.name+"&position=" + parseInt(origin.lat) + "," + parseInt(origin.lng) : ILL_url = "https://native-land.ca/api/index.php?maps=treaties&position=" + parseInt(origin.lat) + "," + parseInt(origin.lng);
```
Okay so since `jshint` [does not take kindly to `eval` operations](https://github.com/jamesallardice/jslint-error-explanations/blob/master/message-articles/eval.md), and since we aren't considering doing a `//jshint ignore:line` here (as that should be the last resort), how about replacing the if ladder with a ternary at the time of declaration?

Reply:
Yes, this would work!!
Do you feel it makes the code a bit to long for one line and not understandable though ?

Reply:
Actually, linting standards do not recommend any line exceeding more than 80 characters, so I guess you can format this line over a few lines, which should comply with the same.

Reply:
Sure, I'll make that change :) Thanks.
But any specific reason why the use of this is better than the use of the if else if block ?

Reply:
Actually I'd like to emphasize more on analyzing the similarity between these lines abstracting them into a function for a much improved code quality (less code, also reusable). The ternary and spread is something I'd like to go with in such situations, but leave it completely on your acumen to decide to keep them or not.